### PR TITLE
Feature/217 reactive form

### PR DESCRIPTION
**Pull request by kitagawa**
Reactive Formの翻訳を行いました。
レビューをお願い致します。

https://github.com/angular/angular-ja/issues/217

### CODE REVIEW

**Comment by kitagawa** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

I signed it!

**Comment by lacolaco** [comment]
Preview ready!

Built with commit fb113a9f0a5f1aaaaa0f74248d826ecbc51edc7b

https://deploy-preview-237--angular-ja.netlify.com

File: aio-ja/content/guide/reactive-forms.md [hunk 1]
```diff
@@ -146,342 +144,345 @@ Generate a `ProfileEditor` component and import the `FormGroup` and `FormControl
...
 </div>
 
-### Displaying form status
+### フォームのステータスを表示する
 
-When you add a required field to the form control, its initial status is invalid. This invalid status propagates to the parent form group element, making its status invalid. Access the current status of the form group instance through its `status` property.
+必須フィールドをフォームコントロールに追加すると、初期ステータスはinvalidになります。invalidステータスは親のフォームグループ要素に伝播し、ステータスがinvalidになります。現在のフォームグループインスタンスのステータスは、`status`プロパティからアクセスします。
 
-Display the current status of `profileForm` using interpolation.
+補完を使用して現在の`profileForm`のステータスを表示させます。
```
Comment (lacolaco) [inline aio-ja/content/guide/reactive-forms.md]: `補間`

File: aio-ja/content/guide/reactive-forms.md [hunk 2]
```diff
@@ -146,342 +144,345 @@ Generate a `ProfileEditor` component and import the `FormGroup` and `FormControl
...
 
 <figure>
   <img src="generated/images/guide/reactive-forms/profile-editor-3.png" alt="Profile Editor Validation">
 </figure>
 
-The **Submit** button is disabled because `profileForm` is invalid due to the required `firstName` form control. After you fill out the `firstName` input, the form becomes valid and the **Submit** button is enabled.
+`profileForm`が`firstName`フォームコントロールの必須によってinvalidになっているため、**Submit** ボタンは非活性になっています。`firstName`を入力すると、フォームはvalidになり　**Submit** ボタンは活性になります。
 
-For more on form validation, visit the [Form Validation](guide/form-validation) guide.
+フォームバリデーションについて詳細は、[Form Validation](guide/form-validation)ガイドを参照してください。
```
Comment (lacolaco) [inline aio-ja/content/guide/reactive-forms.md]: `Form Validation` => `フォームバリデーション`

File: aio-ja/content/guide/reactive-forms.md [hunk 3]
```diff
@@ -146,342 +144,345 @@ Generate a `ProfileEditor` component and import the `FormGroup` and `FormControl
...
-Add the template HTML below after the `<div>` closing the `formGroupName` element.
+下のテンプレートHTMLを`formGroupName`要素の`<div>`閉じタグの後に追加してください。
 
-<code-example path="reactive-forms/src/app/profile-editor/profile-editor.component.html" region="formarrayname" linenums="false" title="src/app/profile-editor/profile-editor.component.html (aliases form array template)">
+<code-example path="reactive-forms/src/app/profile-editor/profile-editor.component.html" region="formarrayname" linenums="false" title="src/app/profile-editor/profile-editor.component.html (エイリアスフォーム配列テンプレート)">
 
 </code-example>
 
-The `*ngFor` directive iterates over each form control instance provided by the aliases form array instance. Because form array elements are unnamed, you assign the index to the `i` variable and pass it to each control to bind it to the `formControlName` input. 
+`*ngFor`ディレクティブはエイリアスフォーム配列インスタンス内の各コントロールインスタンスをイテレートします。フォーム配列要素は名前がついていない為、`i`変数にインデックスを割り当て、各コントロールに渡して`formControlName`にバインドします。
```
Comment (lacolaco) [inline aio-ja/content/guide/reactive-forms.md]: `イテレートします` => `反復します`

**Comment by lacolaco** [comment]
長かったと思いますが、翻訳ありがとうございます！いくつかの部分で修正をお願いします :bow:
また、古い翻訳のファイルが `reactive-form.md.old` にあるかと思いますが、こちらのファイルは削除してもらえますか？

**Comment by kitagawa** [comment]
@lacolaco レビューありがとうございます！
誤字が多くてすいません。。
レビューの指摘箇所の修正と、古い翻訳ファイルの削除致しました。

**Comment by lacolaco** [comment]
修正ありがとうございました！mergeさせていただきます！

**Pull request merged**